.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Bug Report
+about: Create a report to help us improve this project
+
+---
+
+<!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks!
+-->
+
+
+**What happened**:
+
+**What you expected to happen**:
+
+**How to reproduce it**:
+
+**Anything else we need to know?**:
+
+**Environment**:
+- CSI Driver version:
+- Kubernetes version (use `kubectl version`):
+- OS (e.g. from /etc/os-release):
+- Kernel (e.g. `uname -a`):
+- Install tools:
+- Others:
```

Comment:
The environment section doesn't seem to make a whole lot of sense for csi-release-tools?